### PR TITLE
Show ClamAV version and databases versions

### PR DESCRIPTION
**Pull request by kekek2**

### CODE REVIEW

File: security/clamav/src/opnsense/mvc/app/controllers/OPNsense/ClamAV/Api/ServiceController.php [hunk 1]
```diff
@@ -163,4 +163,37 @@ public function reconfigureAction()
             return array("status" => "failed");
         }
     }
+
+    /**
+     * get ClamAV and bases versions
```
Comment (mimugmail) [inline security/clamav/src/opnsense/mvc/app/controllers/OPNsense/ClamAV/Api/ServiceController.php]: and signature versions

File: security/clamav/src/opnsense/mvc/app/views/OPNsense/ClamAV/general.volt [hunk 2]
```diff
@@ -32,6 +32,11 @@ POSSIBILITY OF SUCH DAMAGE.
     <div style="margin-top: 8px;">{{ lang._('No signature database found, please download before use. The download will take several minutes and this message will disappear when it has been completed.') }}</div>
 </div>
 
+<ul class="nav nav-tabs" data-tabs="tabs" id="maintabs">
+    <li class="active"><a data-toggle="tab" href="#general">{{ lang._('General') }}</a></li>
+    <li><a data-toggle="tab" href="#baseinfo">{{ lang._('Viruses base info') }}</a></li>
```
Comment (mimugmail) [inline security/clamav/src/opnsense/mvc/app/views/OPNsense/ClamAV/general.volt]: Just "Versions" ... Viruses base info sounds a bit weird

File: security/clamav/src/opnsense/service/conf/actions.d/actions_clamav.conf [hunk 3]
```diff
@@ -33,3 +33,9 @@ command:/usr/local/opnsense/scripts/OPNsense/ClamAV/freshclam.sh
 parameters:%s
 type:script_output
 message:Check or install signatures
+
+[version]
+command:/usr/local/bin/clamconf | grep "Version:\|main.cvd: version\|.cld: version\|Total number of signatures:"
+parameters:
+type:script_output
+message:Get ClamAV and bases versions
```
Comment (mimugmail) [inline security/clamav/src/opnsense/service/conf/actions.d/actions_clamav.conf]: and signature versions

**Comment by fichtner** [comment]
Merged, thanks!

**Pull request merged**